cmd/ocm-backplane/login/login.go

```diff
@@ -149,7 +150,27 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	if err != nil {
 		return err
 	}
-	logger.Debugf("Backplane Config File Contains: %v \n", bpConfig)
+	// Log non-sensitive fields of BackplaneConfiguration
+	loggableBpConfig := map[string]interface{}{
```

Comment:
seems we removed the AssumeInitialArn from the struct?
would be better if the just rewrite the value if `Govcloud` is true?

like:
```
if bpConfig.Govcloud {
  bpConfig.AssumeInitialArn = "REDACTED"
}
```
so that we do not need to build the new struct?